linux_files/profile

```diff
@@ -45,6 +45,10 @@ export LIBGL_ALWAYS_INDIRECT=1
 # speed up some GUI apps
 export NO_AT_BRIDGE=1
 
+# alias our helper apps
+alias setup='bash /etc/setup'
+alias helpme='bash /etc/setup'
```

Comment:
This seems to be wrong... also it seems that use `setup` as the command might not be good as it might overwrite some of the command with `setup` command

Reply:
os-setup and os-help then?

Reply:
I think that might work.